content_sync/apis/github.py

```diff
@@ -3,30 +3,34 @@
 from base64 import b64decode
 from typing import Iterator
 
-import yaml
 from django.conf import settings
 from github import ContentFile, Github, GithubException, InputGitTreeElement
 from github.Branch import Branch
 from github.Commit import Commit
 from github.InputGitAuthor import InputGitAuthor
 from github.Repository import Repository
 
+from content_sync.api import ContentFileSerializerFactory
 from content_sync.decorators import retry_on_failure
 from main import features
-from ocw_import.api import convert_data_to_content
 from users.models import User
-from websites.models import WebsiteContent
+from websites.models import Website, WebsiteContent
+from websites.site_config_api import SiteConfig
 
 
 log = logging.getLogger(__name__)
 
 
+def decode_file_contents(content_file: ContentFile) -> str:
+    return str(b64decode(content_file.content), encoding="utf-8")
+
+
 class GithubApiWrapper:
     """
     Github API wrapper class
     """
 
-    def __init__(self, website: WebsiteContent):
+    def __init__(self, website: Website, site_config: SiteConfig):
```

Comment:
Instead of passing in a SiteConfig, should be able to get the site config from the website, right? `SiteConfig(website.starter.config)`